### PR TITLE
wrap content below carousel on homepage in a .container div, added in…

### PR DESCRIPTION
**Pull request by matgargano**
… mobile styles to make indenttext3 not indent on mobile, removed indenttext3 as a proof of concept

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -325,3 +325,15 @@ height:325px;
...
 }
 
+
+/*
+
+Mobile Specific Styles
+
+*/
+
+@media only screen and (max-width: 768px) {
```
Comment (matgargano) [inline css/style.css:335]: @bingmo92 i added these mobile specific styles here, which I think work better on mobile.

File: index.html [hunk 2]
```diff
@@ -143,13 +143,15 @@ <h5>Priority L train subway Model</h5>
                         </a>
                 </div>
                 <div class="homebackground">
+                  <div class="container">
```
Comment (matgargano) [inline index.html:146]: @bingmo92  I wrapped everything underneath `.homebackground` in a container, take a look at what bootstrap's css file defines for `.container` to see what happens when we do this

File: index.html [hunk 3]
```diff
@@ -143,13 +143,15 @@ <h5>Priority L train subway Model</h5>
...
+                <p class="">Our Priority Fold has a lightweight, stripped down, simple but robust design, like our previous products, lowering testing costs. The bike can fold easily, allowing customers to save space at home in their apartment or when taking a crowded train. The accessories will be sold separately, allowing a cheap initial purchase and later customization when the customer feels comfortable with it.</p>
                 <hr class="line">
                 <h1>Join the fun!</h1> 
                 <h2 class="indenttext">Participate in alpha and beta testing!</h2>
                 <p class="indenttext3"> Alpha testing consists of trying out the base model of the Priority Fold without any attachment. The expected trial period will run for 3 weeks once invited to participate. Beta testing consists of testing the advanced model of the Priority Prototype Fold with attachments (<a href="about.html">Click here for attachment info!</a>). The expected trial period will run for 3 weeks and once successfully completed you become eligible for early access for the prototype model.</p>
+                  </div>
```
Comment (matgargano) [inline index.html:154]: @bingmo92  this is the closing tag for the `div.container` i added above

File: index.html [hunk 4]
```diff
@@ -143,13 +143,15 @@ <h5>Priority L train subway Model</h5>
                         </a>
                 </div>
                 <div class="homebackground">
+                  <div class="container">
                 <h1>The Priority Fold</h1>
                 <h2 class="indenttext">A Priority Bike Product</h2>
-                <p class="indenttext3">Our Priority Fold has a lightweight, stripped down, simple but robust design, like our previous products, lowering testing costs. The bike can fold easily, allowing customers to save space at home in their apartment or when taking a crowded train. The accessories will be sold separately, allowing a cheap initial purchase and later customization when the customer feels comfortable with it.</p>
+                <p class="">Our Priority Fold has a lightweight, stripped down, simple but robust design, like our previous products, lowering testing costs. The bike can fold easily, allowing customers to save space at home in their apartment or when taking a crowded train. The accessories will be sold separately, allowing a cheap initial purchase and later customization when the customer feels comfortable with it.</p>
```
Comment (matgargano) [inline index.html:149]: it's personal preference, but I don't like .indentext3 here... though I am not a designer ;) @bingmo92

**Review by bingmo92 — APPROVED** [review]

**Review by bingmo92 — APPROVED** [review]

**Review by bingmo92 — APPROVED** [review]

**Review by bingmo92 — APPROVED** [review]

**Review by bingmo92 — APPROVED** [review]

**Review by bingmo92 — APPROVED** [review]

**Pull request merged**